dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/unternehmensfuehrungGovernance/unternehmensrichtlinien/GdvUnternehmensfuehrungGovernanceUnternehmensrichtlinien.kt

```diff
@@ -0,0 +1,10 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.unternehmensfuehrungGovernance.unternehmensrichtlinien
+
+/**
+ * The data-model for the Unternehmensrichtlinien section
+ */
+data class GdvUnternehmensfuehrungGovernanceUnternehmensrichtlinien(
+    val veroeffentlichteUnternehmensrichtlinien: List<String>?,
+    val weitereVeroeffentlicheUnternehmensrichtlinien: String?,
```

Comment:
Why do we have those two? What is the difference between veröffentliche Richtlinien and weitere veröffentliche Richttlinien?

Reply:
The first one is a multi-select-field in the frontend.  So you choose from a pre-defined set of options.
The second one is meant to give you the possibility to give info on further guidelines that are not listed in the first one.

This makes more sense if you check it out in the actual frontend.



The thing is, that we currently handle multi-select-fields as "List<String>" all across Dataland, instead of smth like "List<MultiSelectOptions>".   This is not so good, but I didn't want to change that in this story.

Reply:
Ah sorry, should have looked at the types.. I was just very confused by the names